app/models/project.rb

```diff
@@ -13,13 +13,17 @@ class Project < ActiveRecord::Base
 
   after_create :create_template_defaults
 
-  attr_accessible :name, :description, :template, :archived
+  attr_accessible :name, :description, :template, :archived, :extensions
 
   scope :active,
     :include    => [:users, {:stages => [:hosts, :recipes]}, :configuration_parameters, :user_project_links],
     :conditions => {:archived => false},
     :order      => "projects.name ASC"
 
+  serialize :extensions
```

Comment:
Why did you decide to serialize extensions instead of creating a join? (I'm genuinely curious about the thinking that went into the choice. It's not necessarily wrong.)